core/src/main/java/io/opencensus/tags/TagContext.java

```diff
@@ -41,6 +42,38 @@
   public abstract Iterator<Tag> unsafeGetIterator();
 
   /**
+   * Returns true iff the other object is an instance of {@code TagContext} and contains the same
+   * key-value pairs. Implementations are free to override this method to provide better
+   * performance.
+   */
+  @Override
+  public boolean equals(Object other) {
+    if (!(other instanceof TagContext)) {
+      return false;
+    }
+    HashMultiset<Tag> tags1 = HashMultiset.create(Lists.newArrayList(unsafeGetIterator()));
```

Comment:
I think unsafeGetIterator() has been replaced by iterator()?

Reply:
I made a PR for that change (#605), but it hasn't been merged yet.  If it is approved, then I can just replace all uses of unsafeGetIterator() with iterator().